Masonry/NSArray+MASAdditions.h

```diff
@@ -10,6 +10,11 @@
 #import "MASConstraintMaker.h"
 #import "MASViewAttribute.h"
 
+typedef NS_ENUM(NSUInteger, MASAxisType) {
+    MASAxisTypeHorizon,
```

Comment:
This should read `MASAxisTypeHorizontal` to be consistent with the vertical entry.